src/chemistry/aerosol/cloud_aqueous_chemistry.F90

```diff
@@ -0,0 +1,894 @@
+!----------------------------------------------------------------------------------
+! Cloud aqueous chemistry
+!----------------------------------------------------------------------------------
+module cloud_aqueous_chemistry
+
+  use shr_kind_mod, only : r8 => shr_kind_r8
+  use cam_logfile,  only : iulog
+  use physics_buffer,only: physics_buffer_desc, pbuf_get_index, pbuf_add_field, dtype_r8
+  use physics_types,     only: physics_state
+
+  implicit none
+
+  private
+  public :: sox_inti, setsox
+  public :: has_sox
+
+  logical            ::  inv_o3
+  integer            ::  id_msa
+
+  integer :: id_so2, id_nh3, id_hno3, id_h2o2, id_o3, id_ho2
+  integer :: id_so4, id_h2so4
+
+  logical :: has_sox = .true.
```

Comment:
this should have a description of what it means and how it is used

Reply:
Mary or Francis will have a better idea on this